os/include/assert.h

```diff
@@ -89,7 +89,7 @@ extern char assert_info_str[CONFIG_STDIO_BUFFER_SIZE];
 	{ if (!(f))	{ \
 			  snprintf(assert_info_str, CONFIG_STDIO_BUFFER_SIZE, fmt, ##__VA_ARGS__); \
 			  up_assert((const uint8_t *)__FILE__, (int)__LINE__); \
-		} \
+			} \
```

Comment:
This macro has something wrong indentation. Let's modify it together with new commit. To split different change, could you revert this change from this commit?

Reply:
Done